Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,462 @@
+
+    \textit{Projected End Date: Week 9 (10 if schedule slip occurs)}
+
+\subsubsection{Data Analysis}
+\label{subsubsec: data analysis}
+    This phase is concerned with the analysis of the data collected from the CGT testing. The polynomial approximation will be completed during this
+    phase for the minimum length nozzle. It is also the phase where the performance analysis takes place. This will inlcude the ultimate calculation of the specific impulse
+    of the thuster and if it attains the values we need to achieve the necessary velocity change for orbital decay to initiate atmospheric reentry.
+
+    \textit{Projected End Date: Week 14 or 15 (not terribly important since the team does not theoretically need to be at RIT to conduct this analysis)}
+
+\subsubsection{Final Report}
+\label{subsubsec: final report}
+    At the end of this project a final report will be made to report the outcome. It will outline the design process, fabrication process, and testing process.
+    It will also include the nozzle theory used in the calculations along with all the relevant assumptions and exterior information as needed. The report will provide
+    all the test data and data analysis results. The final product for the report will be the discussion of the polynomial curve-fit. This will be an extremely useful tool
+    for future similar cold gas propulsion systems and must be discussed in depth with an example of how it will be used.
+
+    Once this paper is complete it will be given to the advisors of the SPEX research group for review and then further publications will be discussed. The paper would
+    be great for job interviews to display the work you have done in a formal manner, and one that the interviewer can refer back to at a later date.
+
+    \textit{Projected End Date: End of winter break (the team does not need to be at RIT to write this and they can take advantage of the facilites for testing while they reside there)}
+
+This concludes the milestones for this project.
+
+\section{Externalities}
+  % Things not directly related to the work or outcomes, but related to the project as a whole.
+\subsection{Prerequisite Skills}
+  % Which skills do team members need to have before work can start (not including skills that will be learned ``on the job'')?
+
+Creating a CGT will require a few skills, many of which can be learned throughout the project. The design process will require knowledge of systems and how they
+work together, basic nozzle theory, computer-aided design (CAD), basic electrical engineering, basic coding abilities, and machining. The majors that would
+be most qualified for satisifing these requirements are mechanical engineering \& technology and electrical engineering \& technology. But this project is certainly
+open to all majors because most of these skills can be taught in at least a basic manner.
+
+The basic nozzle theory required here is laid out in several documents
```

Comment:
This section, while helpful is not necessary. Also, most readers will not know or have access to google drive.